webapi/src/main/twirl/queries/sparql/v1/searchExtended.scala.txt

```diff
@@ -27,14 +27,16 @@
  * Performs an extended search. The number of rows returned per matching resource is the product of the number of
  * values that each search criterion matched in the resource.
  *
+ * If the triplestore type is GraphDB, the generated SPARQL is optimised by taking advantage of forward-chaining
+ * RDFS inference.
+ *
```

Comment:
Please put more information here:
- whenever the SPARQL is generated depending on the variable `@triplestore` and it is `graphdb`, inference is used
- however, we are only using inference to do certain things (getting subclasses and subproperties of the explicitly given class or property as well), but when we want to know which property actually macthed, we have to turn of inference in order to know ("GRAPH <http://www.ontotext.com/explicit>")
- if inference is not used (fuseki), we still have to simulate inference using `subclassof` or `subpropertyOf` and a property path expression + or *